msg/tools/uorb_rtps_message_ids.yaml

```diff
@@ -267,18 +267,12 @@ rtps:
     id: 117
   - msg: sensor_accel_fifo
     id: 118
-  - msg: sensor_accel_status
-    id: 119
```

Comment:
@dagar can you rearrange the ID's so we don't have unused numbers? Thanks!

Reply:
Shift the entire set after 118 down? What's the expectation/requirement for these id numbers (persistence, etc)?